activemodel/lib/active_model/error.rb

```diff
@@ -100,6 +105,45 @@ def self.generate_message(attribute, type, base, options) # :nodoc:
       I18n.translate(key, **options)
     end
 
+    def self.generate_custom_message(attribute, type, base, options) # :nodoc:
```

Comment:
Note that this method is a copy of the `generate_message` method. They behave almost exactly the same but I thought I copy it first, validate it's usefulness and maybe extract it later if needed